### PR TITLE
Adding Recursive Objects Breaks Design 

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. Adding an additional object to an object produces an easy-to-use view
2. Reduced left-padding
3. Data is organized and easy to read

### Tasks
- None

### Additional Notes
- None

### CODE REVIEW

**Comment by kylecesmat** [comment]
:star:

**Comment by paulstatezny** [comment]
:star2:

**Comment by paulstatezny** [comment]
:snowflake: :arrow_up:

**Comment by natestormer** [comment]
:hand:

File: application/ui/scss/components/input/_array-param.scss [hunk 1]
```diff
@@ -39,15 +27,91 @@
...
     }
+
+    .array__options td {
+        padding-right : 0;
+    }
+
+    .array__options__inner {
+        border-left   : 2px solid $themeLGray;
+        border-bottom : 10px solid $white;
+        background: rgba(0,0,0,.05);
```
Comment (natestormer) [inline application/ui/scss/components/input/_array-param.scss]: alignment

**Comment by natestormer** [comment]
:pray:

**Comment by dmmcinty** [comment]
:hand:

**Comment by dmmcinty** [comment]
:+1:

**Pull request merged**